css/index.css

```diff
@@ -129,18 +225,162 @@ p {
 footer {
     width: 100%;
     background: black;
+    padding-bottom: 5%;
 }
 
 footer nav {
     width: 60%;
     display: flex;
     justify-content: space-between;
-    align-items: center;
+    align-items:center;
     padding: 20px 2%;
-    font-size: 14px;
+    font-size: 1.4rem;
 }
 
 footer nav a {
     color: white;
     text-decoration: none;
-}
\ No newline at end of file
+}
+
+/*-----------------media queries----*/
+
+@media(max-width:768px){
+
+.container{
+    width: 100%;
+    max-width: 768px;
+    padding:0%;
+}
+header{
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+.logo{
+    padding-top:3%;
+}
+nav{
+    padding-top:4%;
+    width:97%;
+}
+.box{
+    font-size: 1.6rem;
+}
+
+#box1 {
+color: white;
+background:teal;
+}
+
+
+#box3 {
+color: white;
+background:gold;
+}
+
+#box5 {
+color: white;
+background:cadetblue;
+}
+
+#box7 {
+color: white;
+background:coral;
+}
+#box9{
+color: white;
+background:crimson;
+}
```

Comment:
Again the `color` attribute here could be added to the box class above.

You should also focus on your _whitespace_ and syntax. Nested classes typically follow
```
@media (max-width: 720px) {
     #id {
          attribute: value;
     }
     .class {
          attribute: vallue;
     }
}
```